lib/src/web_socket/browser/sockjs.dart

```diff
@@ -34,20 +33,20 @@ abstract class SockJSWebSocket extends CommonWebSocket implements WebSocket {
     // which is community-supported and fully-featured. But, it requires that
```

Comment:
#nit this comment is a little out of date

>The SockJS wrapper library is preferred [over the SockJS Dart port]

But the port is no longer an option, so we might want to reword this